<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>0</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 italic 25pt &quot;Lucida Sans&quot;;
color: rgb(0, 0, 127)</string>
    </property>
    <property name="text">
     <string>Passwerify</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>140</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;MV Boli&quot;;
color: rgb(170, 0, 0)</string>
    </property>
    <property name="text">
     <string>BAD MOMENTS:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;MV Boli&quot;;
color: rgb(0, 170, 0)</string>
    </property>
    <property name="text">
     <string>GOOD MOMENTS:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_security_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Palatino Linotype&quot;;
color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Security level:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_password_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 18pt &quot;Segoe UI Black&quot;;
color: rgb(0, 85, 255)</string>
    </property>
    <property name="text">
     <string>PASSWORD:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_password_value">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;MS Sans Serif&quot;;
color: rgb(53, 70, 75)</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_security_value">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textedit_good_moments">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>361</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MV Boli&quot;;
color: rgb(0, 170, 0)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textedit_bad_moments">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>190</y>
      <width>351</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MV Boli&quot;;
color: rgb(170, 0, 0)</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>711</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="label_error">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>590</y>
      <width>541</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;OCR A Extended&quot;;
color: rgb(170, 0, 0)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_grade_image">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>510</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="button_new_password">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 12pt &quot;Rockwell Extra Bold&quot;;
color: rgb(53, 70, 75)</string>
    </property>
    <property name="text">
     <string>CREATE NEW PASSWORD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_fix_password">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>520</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 12pt &quot;Rockwell Extra Bold&quot;;
color: rgb(53, 70, 75)</string>
    </property>
    <property name="text">
     <string>FIX CURRENT PASSWORD</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
